spec/features/user_edits_account_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+feature 'user edits account', %Q{
+
+  As a registered user
```

Comment:
what happens if I attempt to update my profile with invalid data? There should be a scenario that accounts for that.